src/anomalib/engine/engine.py

```diff
@@ -406,7 +406,7 @@ def _setup_transform(
 
     def _setup_anomalib_callbacks(self) -> None:
         """Set up callbacks for the trainer."""
-        _callbacks: list[Callback] = []
+        _callbacks: list[Callback] = [RichProgressBar(), RichModelSummary()]
```

Comment:
This change is not directly related to the bug but I am now using `RichProgressBar` in the trainer. I don't mind if we drop this as it is not necessary